### ISSUE TITLE
definition of R^2 incorrect

### ISSUE THREAD

**Issue by BethanyL** [deephyper/deephyper#17]
R^2 is defined in deephyper/deephyper/search/nas/model/train_utils.py. For multi-variate regression (predicting a vector output), this definition is incorrect. It sums over both dimensions (# of examples and # of responses/outputs), producing one R^2 score. This is equivalent to flattening the data before calculating R^2. We should change this to match the default behavior of scikit-learn: calculate one R^2 score for each response/output, then average those R^2 scores.

**Comment by Deathn0t** [comment]
Hi @BethanyL , thanks for reporting this issue which was not identified, I am working on the multivariate case on a specific branch: [feature/pblp](https://github.com/deephyper/deephyper/tree/feature/pblp/deephyper)
I started integrating the corrected r2 implementation for multivariate [here](https://github.com/deephyper/deephyper/blob/feature/pblp/deephyper/search/nas/model/train_utils.py#L50)
I started testing it with different training case without comparing to scikit-learn results (to check for consistency) yet: [folder](https://github.com/deephyper/deephyper/tree/feature/pblp/tests/deephyper/benchmark)
I use the following files for testing purposes:
```
multi_loss_test.py
single_loss_multi_var.py
single_loss_test.py
```

**Comment by Deathn0t** [comment]
Resolved in [version 0.1.2](https://github.com/deephyper/deephyper/tree/0.1.2)
